worker/src/main.rs

```diff
@@ -102,17 +106,19 @@ fn main() {
     let sync_block_gossiper = Arc::new(SyncBlockGossiper::new(tokio_handle.clone(), worker));
     let node_api_factory = Arc::new(GlobalUrlNodeApiFactory::new(config.node_url()));
     let direct_invocation_watch_list = Arc::new(WatchListService::new());
+    let enclave = Arc::new(enclave_init().unwrap());
```

Comment:
Made the enclave an `Arc<>` because we need to clone it many times and since it internally holds a `SgxEnclave` object, cloning itself is not cheap. In addition, it also allows us to share the state of the `Enclave` object (if we ever need it)